README.md

```diff
@@ -26,10 +26,17 @@ The [scripts](scripts) folder contains scripts for many frequent tasks:
 
 ### Prerequisite
 
-All the containers need to be started before the tests can be run/debug in visual studio
+All the containers need to be started before the tests can be run/debug in Visual Studio.
 
-* `container.sh start`: starts all the containers.
-* `container.sh stop`: stops all the containers.
```

Comment:
Why did you change the readme? The `container.sh start` is wrapper that starts all the containers w/o the need to start each service individually. May be we should keep both in case somebody wants start one or more vs all.